assets/ports_lists.json

```diff
@@ -7033,7 +7033,7 @@
             "description": "BitTorrent part of full range of ports used most often",
             "udp": true,
             "status": "Unofficial",
-            "port": "6881\u00e2\u20ac\u201c6887",
+            "port": "6881-6887",
```

Comment:
If this is a list of ports will it find 6882 or 6887 and show this description?, Because I am guessing we are using the key of the map and we will find only for port 6881

Reply:
I don't even know why there is this weird unicode symbol. So it's better not to use hyphen in replace of this. I have used [ports.json](https://github.com/mephux/ports.json) file from a github repo. I guess it's a bit old and we can instead use some other good source maybe [port-list](https://github.com/maraisr/ports-list) would work fine.

Reply:
The description of the port is much smaller and less detailed.
But I guess this is go for our us case.

The best option is to find a dart package for that but I didn't find any.

Reply:
I'll see this port description in next PR. I've create an issue #8, so that we don't ignore it.